### ISSUE TITLE
Bug: Accordion headings not focusable in IE

### ISSUE THREAD

**Issue by robertscholz** [angular-ui/bootstrap#3063]
Accordion headings have anchor tags with empty href attribute that are not focusable in Internet Explorer, which makes it impossible to use keyboard navigation. A possible fix is to set href value to **javascript:;** or **javascript:void(0);** as discussed [here on SO] (http://stackoverflow.com/questions/5637969/is-an-empty-href-valid).

**Comment by rvanbaalen** [comment]
If I'm not mistaken, this is fixed since https://github.com/angular-ui/bootstrap/commit/9ca4ec399be0e0e8f6c3fe6fd924a7d94ce669b5
